Extensions/3D/JsExtension.js

```diff
@@ -800,6 +800,20 @@ module.exports = {
         .markAsSimple()
         .setHidden()
         .setFunctionName('hasAnimationEnded');
+
+      object
+        .addScopedAction(
+          'SetCrossfadeDuration',
+          _('Set cross fade duration'),
+          _('Set the duration of the cross fading between two animations.'),
+          'Set cross fading duration of _PARAM0_ to _PARAM1_',
```

Comment:
I think in english it's a single word usually:

```suggestion
          _('Set crossfade duration'),
          _('Set the crossfade duration when switching to a new animation.'),
          'Set crossfade duration of _PARAM0_ to _PARAM1_',
```